### ISSUE TITLE
Midpoint Review - IA Feedback - Ask VA Integration - Ask VA

### ISSUE THREAD

**Issue by erinrwhite** [department-of-veterans-affairs/va.gov-team#71084]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] Comment on this ticket:
  - [x] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [x] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [x] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [x] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- Thank you for your work to bring this form into the VA.gov experience!
- During our meeting I had questions/concerns about the length of the user flow. After reviewing the artifacts, revisiting feedback from the Design Intent, and hearing future plans for the service, I am less concerned about this. Thank you for including questions throughout the research plan that will help us understand users' expectation of the flow!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

#### Search by reference number

A couple of things on this one.
 - "View details" link opens a modal, which then sends the user to sign in. We generally try to limit the use of modals. Can the link text be updated to "Sign in to check details" and skip the modal?
 - The "View details" can be easy to miss - is there a way to change the visual presentation so that it is more  closely connected with the information above, and visible as an action to take? The [Card](https://design.va.gov/components/card) pattern wouldn't be quite appropriate since there's only one item. But maybe some other visual grouping of these items.
![Screenshot 2023-12-04 at 9 56 06 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/755248/0e87b003-6479-46bb-9d68-ae1ccd34d6cc)

#### Dashboard: Business tab card details

On the dashboard view "Business" tab, would it be useful for the name of the Veteran to be displayed on the cards? I am assuming that someone who submits questions for others as part of their work may be working with multiple Veterans at the same time, so being able to tell which question applies to which Veteran might be useful.

**Comment by tygindraux** [comment]
Hi Erin (@erinrwhite)! Thanks so much for your feedback, and for spurring the discussion on Friday about our plans to simplify the form more over time. I'm glad to read that your concerns are lessened!

Your feedback totally makes sense. We'll plan to review and make changes this week, and then I'll comment again to let you know how that went. Thanks again.

**Comment by tygindraux** [comment]
Hi @erinrwhite, we've finished updates based on your feedback. These included:
- search by ref number: updating the link from 'view details' to 'sign in to check details' and removing the modal
- search by ref number: updating the styling of 'sign in to check details' so it's more visible (see screenshot attached), we still plan to monitor this later in testing

We also considered your feedback about including the name of the Veteran in the display of the cards (within business inquiries, in the dashboard). Here's why we're not suggesting to add this (at least, for now):
- not all business inquiries will be tied to a Veteran, for example, they might ask, "I need a debt list for the following schools: Washington State University 23980, San Francisco..."
- currently, the question isn't included in the dashboard, and the reason we brought this in (and made it quite prominent) is because we learned that business users often browse their inquiries by question, because they tend to start with the student's name if it's about a student/Veteran
- we're planning to do some more iterative testing with business users (since they make up around 36% of users of Ask VA, currently)
- when testing, our preference is to show them less information so we can see what's missing when they use the prototypes
- if we learn that they're missing information like the Veteran's name, we'll need to think about how to do this (given the range of inquiries)

I hope that helps! I'm going to close this ticket now, but let us know if there's any follow up questions or comments once you have a chance to review this. Thanks again!
![Screen Shot 2023-12-05 at 11 18 40 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10949237/3374416d-1ea8-40a1-964a-35629dc10e11)
